inc/namespace.php

```diff
@@ -427,6 +438,36 @@ function load_object_cache_redis() {
 	wp_cache_init();
 }
 
+/**
+ * Load Redis session handler.
+ *
+ * @return void
+ */
+function load_session_handler() {
+	global $wp_object_cache, $redis_server;
+
+	if ( headers_sent() ) {
+		return;
+	}
+
+	$client = new Predis\Client( $wp_object_cache->build_client_parameters( $redis_server ), [ 'prefix' => 'sessions:' ] );
+	$handler = new Predis\Session\Handler( $client );
+
+	session_set_save_handler( $handler, true );
+	session_name( 'altis_session' );
+	session_start();
```

Comment:
Looks like we don't need to call `session_start()` ourselves based on the slack conversation, it's up to developers to do so. Might be worth extending the documentation page a bit to indicate that and what the default cookie name is.